src/js/components/DataSort/DataSort.js

```diff
@@ -98,15 +98,19 @@ export const DataSort = ({ drop, options, ...rest }) => {
 
   if (!drop) return content;
 
+  const tip = format({
+    id: 'dataSort.open',
+    messages: messages?.dataSort,
+  });
+
   const control = (
     <DropButton
       id={`${dataId}--sort-control`}
-      aria-label={format({
-        id: 'dataSort.open',
-        messages: messages?.dataSort,
-      })}
+      aria-label={tip}
+      tip={tip}
       kind={theme.data.button?.kind}
       icon={<Descend />}
+      hoverIndicator
```

Comment:
```suggestion
```
@abdulbasithqb Based on the [above discussion](https://github.com/grommet/grommet/pull/7069/files#r1440938213), would you be willing to remove `hoverIndicator` from this PR? Thank you! Here's a code suggestion.

Reply:
This will likely require updating the snapshots as well.

Reply:
Done

Reply:
Thank you!